automation/nightly/test-nightly-build.sh

```diff
@@ -37,25 +37,45 @@ latest_cdi_image=$(curl -sL "https://storage.googleapis.com/kubevirt-prow/devel/
 IFS=: read -r cdi_image cdi_tag <<< "${latest_cdi_image}"
 latest_cdi_commit=$(curl -sL "https://storage.googleapis.com/kubevirt-prow/devel/nightly/release/kubevirt/containerized-data-importer/${latest_cdi}/commit")
 
+latest_cnao=$(curl -sL https://storage.googleapis.com/kubevirt-prow/devel/nightly/release/kubevirt/cluster-network-addons-operator/latest)
+latest_cnao_image=$(curl -sL "https://storage.googleapis.com/kubevirt-prow/devel/nightly/release/kubevirt/cluster-network-addons-operator/${latest_cnao}/operator.yaml" | grep "image:" | grep "cluster-network-addons-operator" | sed -E "s|^ +image: (.*)$|\1|")
+IFS=: read -r cnao_image cnao_tag <<< "${latest_cnao_image}"
+latest_cnao_commit=$(curl -sL "https://storage.googleapis.com/kubevirt-prow/devel/nightly/release/kubevirt/cluster-network-addons-operator/${latest_cnao}/commit")
+
 # Update HCO dependencies
 go mod tidy
 go mod vendor
-rm -rf kubevirt cdi
+rm -rf kubevirt cdi cnao
 
 # Get latest kubevirt
 git clone https://github.com/kubevirt/kubevirt.git
-(cd kubevirt; git checkout "${latest_kubevirt_commit}")
+(
+  cd kubevirt
+  git checkout "${latest_kubevirt_commit}"
+ )
 
 # Get latest CDI
 git clone https://github.com/kubevirt/containerized-data-importer.git cdi
-(cd cdi; git checkout "${latest_cdi_commit}")
+(
+  cd cdi
+  git checkout "${latest_cdi_commit}"
+)
```

Comment:
nit: this `)` is indented differently than the others added

Reply:
fixed.